docs/technical_reference/unit_models/membrane_distillation_0D.rst

```diff
@@ -174,24 +201,139 @@ if ``pressure_change_type`` is set to ``PressureChangeType.calculated``:
+
+if ``MD_configuration_type`` is set to ``MDconfigurationType.DCMD``:
+
+.. csv-table::
+   :header: "Description", "Equation"
+   
    "Conduction heat flux across membrane", ":math:`q_{\text{cond}}(t, x) = \frac{k_{\text{m}}}{\sigma} \left( T_{\text{interface}, h}(t, x) - T_{\text{interface}, c}(t, x) \right)`"
+
+if ``MD_configuration_type`` is set to ``MDconfigurationType.PGMD_CGMD``:
+
+.. csv-table::
+   :header: "Description", "Equation"
+   
+   "Conduction heat flux across membrane", ":math:`q_{\text{cond}}(t, x) = \frac{k_{\text{m}}}{\sigma} \left( T_{\text{interface}, h}(t, x) - T_{\text{interface}, gap}(t, x) \right)`"
+   "Conduction heat flux across gap", ":math:`q_{\text{gap}}(t, x) = \frac{k_{\text{m}}}{\sigma} \left( T_{\text{interface}, gap}(t, x) - T_{\text{interface}, c}(t, x) \right)`"
+
+if ``MD_configuration_type`` is set to ``MDconfigurationType.VMD``:
+
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Vapor expansion heat flux", ":math:`q_{\text{exp}}(t, x) = \frac{R \cdot T}{M} \ln\left( \frac{P_f}{P_p} \right) \cdot J(t, x)`"
+
+
+if ``MD_configuration_type`` is set to ``MDconfigurationType.DCMD``:
+
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Mass transfer balance between hot and cold channel", ":math:`\dot{m}_{\text{cold}}(t, x, p, j) = -\dot{m}_{\text{hot}}(t, x, p, j)`"
+
+   "Conductive heat transfer to cold channel", ":math:`q_{\text{cond, hot}}(t, x) = -q_{\text{cond, cold}}(t, x)`"
+
+
+if ``MD_configuration_type`` is set to ``MDconfigurationType.VMD``:
+
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Mass transfer from vapor phase to vacuum channel", ":math:`\dot{m}_{\text{cold}}(t, x, Vap, j) = -\dot{m}_{\text{hot}}(t, x, Liq, j)`"
+
+   "Conductive heat transfer to cold channel", ":math:`q_{\text{cond, hot}}(t, x) = -q_{\text{exp}}(t, x)`"
+
+   "Cold channel inlet temperature", ":math:`T_{\text{cold, in}}(t) = T_{\text{hot, in}}(t)`"
+
+if ``MD_configuration_type`` is set to ``MDconfigurationType.PGMD_CGMD``:
+
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Mass transfer balance between hot and gap channel",  ":math:`\dot{m}_{\text{gap}}(t, x, Liq, H2O) = -\dot{m}_{\text{hot}}(t, x, Liq, H2O)`"
+
+   "Conductive heat transfer between channels", ":math:`q_{\text{cold}}(t, x) = -q_{\text{hot}}(t, x) - ΔH_{\text{hot}}(t, x) - ΔH_{\text{gap}}(t, x)`"
+
+   "Conductive heat transfer across gap",  ":math:`q_{\text{gap}}(t, x) = q_{\text{cold}}(t, x)`"
+
+
+
+.. csv-table::
+   :header: "Description", "Equation"
+
    "Average conduction heat across membrane", ":math:`q_{\text{cond, avg}}(t) = \frac{1}{N} \sum_{x} q_{\text{cond}}(t, x)`"
    "Total permeate production", ":math:`M_p = A \cdot J_{\text{avg}}`"
    "Total conduction heat transfer", ":math:`q_{\text{cond,total}} = - A \cdot q_{\text{cond,avg}}`"
```

Comment:
Unclear whether these are common to all configurations or tied to PGMD_CGMD config when looking solely at the rst file, which displays two separate tables one after the other.

Reply:
addressed